I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 241, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script/script.rpy", line 24, in script
    show text "Rights-Quest" with dissolce
  File "game/script/script.rpy", line 24, in <module>
    show text "Rights-Quest" with dissolce
NameError: name 'dissolce' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 241, in script call
    call _splashscreen from _call_splashscreen_1
  File "game/script/script.rpy", line 24, in script
    show text "Rights-Quest" with dissolce
  File "D:\Downloads\renpy-8.2.1-sdk\renpy\ast.py", line 1359, in execute
    paired = renpy.python.py_eval(self.paired)
  File "D:\Downloads\renpy-8.2.1-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "D:\Downloads\renpy-8.2.1-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script/script.rpy", line 24, in <module>
    show text "Rights-Quest" with dissolce
NameError: name 'dissolce' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.1.24030407
Rights Quest 1 1.0
Wed Apr 10 11:34:01 2024
